### DISCUSSION TITLE
Option for tree-sitter test to skip some tests

### DISCUSSION THREAD

**Discussion by leotrs** [tree-sitter/tree-sitter#1987] [Ideas] [answered]
I'm looking for the opposite of the -f flag: a way to tell `tree-sitter test` to run all except for some tests.

**Comment by amaanq** [comment, marked as answer]
https://tree-sitter.github.io/tree-sitter/creating-parsers/5-writing-tests.html#attributes

Also, the test command takes two flags for including/excluding tests rather than filtering them:
```
  -i, --include <string>: Only run corpus test cases whose name matches the given regex
  -e, --exclude <string>: Only run corpus test cases whose name does not match the given regex
```
